railties/lib/rails/application.rb

```diff
@@ -183,6 +185,13 @@ def message_verifier(verifier_name)
       end
     end
 
+    #
+    def deprecators
+      @deprecators ||= ActiveSupport::Deprecation::Deprecators.new.tap do |deprecators|
+        deprecators[:rails] = ActiveSupport::Deprecation.instance
```

Comment:
I don't think we need a Rails deprecator. Each gem inside the framework would have it own. Unless this is the railties deprecator. Maybe we should have an application deprecator.

Reply:
We want `ActiveSupport::Deprecation.instance` in `Rails.application.deprecators` so that `Rails.application.deprecators.silence {}` and other configuration settings will apply to it too, at least until we remove `ActiveSupport::Deprecation.warn`.

We can use a different name than `:rails` though, such as `:application` or `:default`.

Reply:
Ah, got it. I think `:rails` make sense then.